### PR TITLE
PXC-2728: Release notes - v 5.6.46-28.38

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+on |date|.  Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_ or from our
+:ref:`software repositories <installation>`.
+
+|PXC| |release| is now the current release, based on the following:
+
+- `Percona Server 5.6.46
+  <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.46-86.2.html>`_
+- Codership WSREP API release 5.6.46
+- Codership Galera library 3.28
```
Comment (rameshvs02) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst:17]: Galera library version is 3.38
```
mysql> show status like '%wsrep_provider_version%';
+------------------------+----------------+
| Variable_name          | Value          |
+------------------------+----------------+
| wsrep_provider_version | 3.38(rb3295e6) |
+------------------------+----------------+
1 row in set (0.00 sec)

mysql>

Comment (kennt-percona) [reply doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst:17]: 3.28 is the correct Codership version.  PXC versions are different.

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+All Percona software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- :jirabug:`2684`: A cluster node could hang when trying to access a table which
```
Comment (kennt-percona) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst]: We need to change this from PXC-2684 to PXC-2729 (that is still an open issue).

**Review by tduzan-te — APPROVED** [review]
Looks good to me.

**Pull request merged**